docparse/chunking_strategies.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Chunking Strategies'
+description: 'The chunking strategies supported by Aryn DocParse'
+icon: 'brackets-curly'
+---
+
+
+
+When calling DocParse, you can specify a chunking strategy to the partition_file call. You can enable the default chunking options by specifying an empty dict:
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, chunking_options={})
+```
+
+Here is an example specifying certain chunking options:
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, 
+      chunking_options={
+         "strategy": "context_rich",
+         "tokenizer": "openai_tokenizer",
+         "tokenizer_options": {
+            "model_name": "text-embedding-3-small"
+         },
+         "merge_across_pages": True,
+         "max_tokens": 512,
+      }
+   )
+```
+
+## Options 
+The options that you can specify in the dict include the following: 
+
+ - `strategy`: A string specifying the strategy to use to combine and split chunks. Valid values are `context_rich`, `mixed_multi_column`, and `maximize_within_limit`. The default and recommended chunker is `context_rich` as `{'strategy': 'context_rich'}`.
```

Comment:
'Combine and split chunks" for folks who don't use sycamore may seem confusing since not everyone does merge and split_elements the way we do. Maybe replace with 'strategy to use for chunking' ? Okay either way.

Reply:
makes sense. Will change